x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_table_filters/rules_table_filters.tsx

```diff
@@ -67,7 +67,7 @@ const RulesTableFiltersComponent = ({
 
   const handleSelectedTags = useCallback(
     (newTags) => {
-      if (!isEqual(newTags, selectedTags)) {
+      if (!deepEqual(newTags, selectedTags)) {
```

Comment:
These (and I think a few others) are just `string[]`. Is `deepEqual` still necessary/more performant in those cases?